scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/kafka/Configuration.kt

```diff
@@ -139,10 +141,13 @@ fun getKafkaProperties(params: KafkaStreamsParams): KafkaProperties {
         this[ConsumerConfig.AUTO_OFFSET_RESET_CONFIG] = "latest"
         this[ConsumerConfig.MAX_PARTITION_FETCH_BYTES_CONFIG] = KAFKA_MAX_MESSAGE_BYTES
         this[ConsumerConfig.FETCH_MAX_BYTES_CONFIG] = KAFKA_MAX_MESSAGE_BYTES
+        this[ConsumerConfig.SEND_BUFFER_CONFIG] = KAFKA_MAX_MESSAGE_BYTES
+        this[ConsumerConfig.RECEIVE_BUFFER_CONFIG] = KAFKA_MAX_MESSAGE_BYTES
         this[ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG] = 60000
 
         this[ProducerConfig.LINGER_MS_CONFIG] = 0
         this[ProducerConfig.MAX_REQUEST_SIZE_CONFIG] = KAFKA_MAX_MESSAGE_BYTES
+        this[ProducerConfig.BUFFER_MEMORY_CONFIG] = KAFKA_MAX_MESSAGE_BYTES
```

Comment:
not part of this PR, but why do we need to set steams, producer and consumer configs with the same values?

Reply:
Probably does not further validation only a subset is needed but I assume they are used by different parts of kstreams for setting tings up.